spec/lib/percy/capybara/loaders/base_loader_spec.rb

```diff
@@ -14,6 +14,18 @@
       expect(resource.content).to include('Hello World!')
       expect(resource.sha).to eq(Digest::SHA256.hexdigest(resource.content))
     end
+    it 'includes an iframe resource' do
```

Comment:
Excellent, I'm glad we're adding this test otherwise all RackTests would have broke!

Let's just catch that error and silence it. Yeah?